src/modules/land/utils.ts

```diff
@@ -173,7 +173,7 @@ export function locateNextLand(landTiles: Record<string, LandTile>, currentLandI
   const nextIndex = (((index + 1) % landIds.length) + landIds.length) % landIds.length
 
   const nextLandId = landIds[nextIndex]
-  return landTiles[nextLandId]!.land
+  return landTiles[nextLandId].land
```

Comment:
could `landTiles[nextLandId]` be undefined?

Reply:
I'm not sure, but the result would be the same with or without the `!`. 
This change was required by the linter.